### DISCUSSION TITLE
Stormwatch functionality for Predbat?

### DISCUSSION THREAD

**Discussion by PianSom** [springfall2008/batpred#1603] [Ideas]
The Powerwall has a cool feature that if a storm is headed towards it then it will up its held-back reserves to cope with any potential power outages. One day soon (probably) Winter will actually start around here, and I was wondering if anyone had implemented any similar functionality for Predbat.

I was thinking about a HA automation which monitors a weather sensor and then changes Predbat's SoC reserve (and puts it back after the storm has passed). Rather than build from scratch, I thought I'd ask if anyone has already put together something like this?

**Comment by LynchMicro** [comment]
I would use this. I manually alter the battery keep and battery min levels if there's a Met office yellow wind warning in place. Should be possible to automate

**Reply by gcoan** [reply]
On my to-do list is to investigate the Global Disaster alert service GDACS which could provide the source for such automation https://youtu.be/LcNo4MCQRzM
